packages/components/src/typography/typography.tsx

```diff
@@ -46,6 +47,8 @@ function styleFromColor(color: TypographyColor): TwStyle {
       return tw`text-brand-700`;
     case "info":
       return tw`text-info-700`;
+    case "neutral":
+      return tw`text-neutral-500`;
```

Comment:
Unrelated to your changes in this PR (which LGTM), but since this is just a tailwind classname, why do we need `tw`?

Could we instead do something like

```jsx
return (
  <TypographyComponent
    className={classnames(styleFromColor(color), styleFromSize(size))}
  />
);
```

And have it work the same? It's not clear to me what we're getting out of the macro system.  I'm sure I'm just missing something, though!

---

Like I said, not related to your actual changes.

Reply:
that's a good question -- from my reading, I would say the main benefit of the macro is no extra runtime code. from [twin.macro docs](https://github.com/ben-rogerson/twin.macro#features):
> Adds no size to your build - Twin converts classes you’ve used into css objects using Babel and then compiles away, leaving no runtime code

and there are some other nice things in there like catching mistypings. 

Otherwise, it seems (from [this article](https://www.freecodecamp.org/news/how-to-style-your-react-apps-with-less-code-using-tailwind-css-and-styled-components/#tailwind-css-and-styled-components))
> "It's perfectly fine to use the utility classes with the className attribute, but if you want to nest properties you have to use the twin.macro library"

in that case, we'd add a separate `tailwind.css` file that imports the necessary utilities. I think for the first reason, if we stick with `styled-components` we should definitely leverage the macro. If we switch to Linaria, I'm not as sold on the macro ...

cc @dcwither for other thoughts